app/insomnia/Keyboard-shortcuts.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Keyboard shortcuts for Insomnia application"
+content_type: reference
+layout: reference
+breadcrumbs:
+  - /insomnia/
+
+products:
+  - insomnia
+
+works_on:
+  - insomnia
```

Comment:
```suggestion
```
I think this line was making the[ frontmatter validator cranky](https://github.com/Kong/developer.konghq.com/actions/runs/16496769994/job/46644304387?pr=2316)